tests/v2/validation/provisioning/k3s/custom_cluster_test.go

```diff
@@ -90,12 +93,13 @@ func (c *CustomClusterProvisioningTestSuite) ProvisioningK3SCustomCluster(extern
 	tests := []struct {
 		name      string
 		nodeRoles []string
+		hardening *provisioning.Config
```

Comment:
can you add this var to the example readme so we know how to use it in the config.yaml?

Reply:
Sure thing, that's a good idea. I'll go and make that change.